docs/STATES.md

```diff
@@ -0,0 +1,49 @@
+# States
+
+Components can have various states. We want to document which states exist and their definitions.
+
+### Normal
+
+The state when no other state applies.
+
+### Hover
+
+State when mouse is hovering over the component. Does not exist on touch devices.
+Should visually highlight the component.
+For accessibility compliance, the visual highlight must not be reduced to color, but should either include translation, cursor/pointer change or other effects that make it understandable for color-blind consumers.
+
+### Focused
+
+State when component or element has focus via keyboard.
+Should provide an outline when component has focus (via keyboard).
+
+### Active
+
+State when component or element is active and not a form element.
+(e.g. an active tab in a tab group, an expanded accordion panel, an pressed button)
+Should visually highlight the component.
+For accessibility compliance, the visual highlight must not be reduced to color, but should either include translation, cursor/pointer change or other effects that make it understandable for color-blind consumers.
```

Comment:
Are you sure? Probably we have to add in the documentation which states have to be compared.
E.g. the active state of the button only changes the color (in difference to the base button state). But the difference between hover and active state would include more than color changes.
@mcilurzo